doc/api_reference.py

```diff
@@ -899,6 +900,13 @@ def _get_submodule(module_name, submodule_name):
                     "RandomizedSearchCV",
                 ],
             },
+            {
```

Comment:
Is this file automatically generated? If this is the case, does it makes sense to drop it from `git` because it will be modified whenever we add a new function/classes

Reply:
No it's not. It serves a similar purpose as the previous `classes.rst`.

Reply:
OK, I see. Did we document how to do it somewhere?

Reply:
I think I mentioned in the comments in this file. Maybe I should in addition point to this file in the developer docs.

Reply:
Yep something about it here: https://scikit-learn.org/dev/developers/contributing.html#documentation would be great.